public/js/app/pages/__tests__/__snapshots__/CourseMemo.test.js.snap

```diff
@@ -233,7 +217,7 @@ exports[`Page CourseMemo renders a side menu and a page structure 1`] = `
                   aria-hidden="true"
                   id="page-sub-heading"
                 >
-                  Missing mandatory information
+                  TEST121 undefined 7.5 credits
```

Comment:
Why is `undefined` in the name?

Reply:
If title is missing then it is showing undefined. Now title is passed as empty string